torch/_torch_docs.py

```diff
@@ -1788,6 +1788,35 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.clamp`.
 """.format(**common_args))
 
+add_docstr(torch.column_stack,
+           r"""
+column_stack(tensors, *, out=None) -> Tensor
+
+Stack 1D tensors as columns into a 2D tensor, 2D tensors are stacked just like :func:`torch.hstack`.
```

Comment:
This description is already good, but I think something like this would be a little clearer:

Creates a new 2-dimensional tensor by horizontally stacking the tensors in :attr:\`tensors\`. :

attr:\`tensors\` must be entirely composed of 1-dimensional and 2-dimensional tensors whose first dimension is the same size. The 1-dimensional tensors in :attr:\`tensors\` are reshaped into (*, 1) columns before being horizontally stacked.

Reply:
Not related to the documentation, but we should be sure we error out if a tensor and not a list of tensors is given. NumPy has some odd behavior when given a single array, probably because it's so aggressive at interpreting its inputs:

```
# This seems correct
a = np.array(((1, 2), (3, 4), (5, 6)))
np.column_stack((a,))
:  array([[1, 2],
       [3, 4],
       [5, 6]])

# Maybe technically correct, but confusing
a = np.array(((1, 2), (3, 4), (5, 6)))
np.column_stack((a))
: array([[1, 3, 5],
       [2, 4, 6]])
```

Reply:
Thanks so much for the advice of the docs, the description has been updated accordingly.
And the test case of the single array input is added o make sure the appropriate errors are raised.